src/invidious/views/watch.ecr

```diff
@@ -76,6 +76,19 @@ we're going to need to do it here in order to allow for translations.
 <div class="h-box">
     <h1>
         <%= title %>
+        <span class="length-watch-page"> 
+            <% time_span_length = video.length_seconds.seconds %>
+
+            <% if time_span_length.days > 0 %>
+                <%= time_span_length.days %> d <%= time_span_length.hours %> hr <%= time_span_length.minutes %> min 
+            <% elsif time_span_length.hours > 0 %>
+                <%= time_span_length.hours %> hr <%= time_span_length.minutes %> min 
+            <% elsif time_span_length.minutes > 0 %>
+                <%= time_span_length.minutes %> min
+            <% else %>
+                <%= time_span_length.seconds %> seconds
+            <% end %>
```

Comment:
Can you make a dedicated function in `helpers/utils.cr` for that?
It will be easier to read than a lot of crystal code in the HTML ^^

Reply:
Here it is! https://github.com/discollizard/invidious/commit/2bdd20906cac74e9dcb8afc80e63d621c246c7e8